modules/appeals_api/spec/docs/v2/nod_spec.rb

```diff
@@ -34,7 +32,9 @@
           value: JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_10182_minimum.json')))
         },
         'all fields used' => {
-          value: JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_10182_extra.json')))
+          value: JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_10182_extra.json'))).tap do |data|
+            data.dig('data', 'attributes')&.delete('claimant') unless DocHelpers.wip_doc_enabled?(:nod_v2_claimant)
```

Comment:
Sadly had to use `DocHelpers.wip_doc_enabled?` here instead of just `wip_doc_enabled?` because of the way rspec tries to ensure your included methods are only called in certain contexts.